config/routes.rb

```diff
@@ -1,12 +1,17 @@
 Rails.application.routes.draw do
+  get 'sessions/new'
+
   root "application#hello"
   root "users#index"
   root "static_pages#home"
   get  "static_pages/home"
-  get  "/help",    to: "static_pages#help", as: t("config.settings.routes.helf")
-  get  "/about",   to: "static_pages#about"
+  get  "/help", to: "static_pages#help", as: t("config.settings.routes.helf")
```

Comment:
```suggestion
  get "/help", to: "static_pages#help", as: t("config.settings.routes.helf")
```
k dùng i18n trong routes nhé